portal-web/test/functional/com/liferay/portalweb/paths/portlet/gdpr/ExportPersonalData.path

```diff
@@ -25,6 +25,11 @@
 	<td>//table//tr[contains(.,'${key_application}')]//input[@type='checkbox']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>APPLICATION_TABLE_HEAD_ITEM</td>
```

Comment:
Would you please double check if the exist locator _LexiconTable#TABLE_HEADER_ works? Thanks in advance.

Reply:
Sure. Thank you!